src/Integration.UnitTests/NewConnectedMode/BindingConfigurationTests.cs

```diff
@@ -40,17 +40,29 @@ public void Static_CheckStandaloneSingleton()
         public void StaticCreator_InvalidArgs_NullProject_Throws()
         {
             // Arrange
-            Action act = () => BindingConfiguration.CreateBoundConfiguration(null, SonarLintMode.Connected);
+            Action act = () => BindingConfiguration.CreateBoundConfiguration(null, SonarLintMode.Connected, "c:\\");
 
             // Act & Assert
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("project");
         }
 
+        [DataTestMethod]
```

Comment:
I'm not sure why the `DataTestMethod` attribute exists; all of the existing data-driven tests just use `TestMethod` and they work fine...

Reply:
Ah, I didn't know that, I thought you had to use DataTestMethod.